definitions/npm/axios_v0.15.x/flow_v0.28.x-/axios_v0.15.x.js

```diff
@@ -93,6 +93,7 @@ declare module 'axios' {
       request: AxiosRequestInterceptor<mixed>,
       response: AxiosResponseInterceptor<mixed>,
     };
+    defaults: AxiosXHRConfig<T>
```

Comment:
Based on
https://github.com/deecewan/flow-typed/blob/master/definitions/npm/axios_v0.15.x/flow_v0.28.x-/axios_v0.15.x.js#L83
and
https://github.com/mzabriskie/axios/blob/master/lib/core/Axios.js#L16
I'm compelled to say that `defaults` is an instance of `AxiosXHRConfigBase`
I have no idea about what to put into the type parameter, but based the request and response interceptors I think it might be `mixed`